src/common/Widget.js

```diff
@@ -43,6 +43,25 @@
         this._watchArr = [];
 
         this._renderCount = 0;
+
+        this._hpcc_debug = false;
+        var context = this;
+        var def = window.location.search.split("?")[1];
```

Comment:
we can push it off into a global and check for it ... but splitting a small string shouldn't be heavy,